.circleci/config.yml.j2

```diff
@@ -2,6 +2,8 @@
 {%- set docker_version = '20.10.11' -%}
 ---
 version: 2.1
+orbs:
+  python: circleci/python@2.0.3
```

Comment:
Do we really need this orb? Seems like it's a more complicated way to do `run: pip install requirements.txt`

Reply:
Using Orb has also other benefit especially in case of this kind of build where we have multiple time same execution. Orb allow us to make reusable snippet. 

In our case we have to install pip dependencies to all test phase.

- Without orb, It will re-execute the pip install each test phase.
     - https://app.circleci.com/pipelines/github/astronomer/ap-vendor/922/workflows/5cc002cf-8910-4945-bdc0-85d527ba4fe2/jobs/34146 
     - https://app.circleci.com/pipelines/github/astronomer/ap-vendor/922/workflows/5cc002cf-8910-4945-bdc0-85d527ba4fe2/jobs/34149
- With orb, it will execute the pip install first with docker image test and then it will re-use it to all other.
    - https://app.circleci.com/pipelines/github/astronomer/ap-vendor/920/workflows/d781b5fa-97fa-4852-a03b-541abc7304f6/jobs/34132
    -  https://app.circleci.com/pipelines/github/astronomer/ap-vendor/920/workflows/d781b5fa-97fa-4852-a03b-541abc7304f6/jobs/34135

Here we don't have more package to install for now, so will be not much difference into execution time, but still it is suggested to use.

Reply:
Man, they should update [their docs](https://circleci.com/developer/orbs/orb/circleci/python). They don't list any of those details.